src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { ChangeEvent, MouseEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const todos = todosFromServer.map(todo => ({
```

Comment:
```suggestion
const initialTodos = todosFromServer.map(todo => ({
```
consider more meaningful naming